common_utilities/generate_output_path.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+from common_utilities.path_settings import PathSettings
+
+
+def generate_output_path():
+    OUTPUT_PATH = "OutputFiles/"
+    output_path = (os.path.abspath(os.path.join(PathSettings.BASE_DIR, OUTPUT_PATH)))
+    print('Path not present')
```

Comment:
should this print statement be inside the if block?

Reply:
updated : https://github.com/dimagi/dimagi-qa/pull/145/commits/8746ae0ca107fe7dae22690d90b0d8b6c3c49545